src/main/java/dk/cs/aau/huppaal/controllers/RunConfigurationEditorController.java

```diff
@@ -0,0 +1,40 @@
+package dk.cs.aau.huppaal.controllers;
+
+import com.google.gson.Gson;
+import com.jfoenix.controls.JFXButton;
+import com.jfoenix.controls.JFXListView;
+import com.jfoenix.controls.JFXRippler;
+import dk.cs.aau.huppaal.BuildConfig;
+import dk.cs.aau.huppaal.HUPPAAL;
+import dk.cs.aau.huppaal.runconfig.RunConfiguration;
+import dk.cs.aau.huppaal.runconfig.RunConfigurationPreferencesKeys;
+import javafx.fxml.Initializable;
+import javafx.scene.layout.GridPane;
+
+import java.net.URL;
+import java.util.List;
+import java.util.ResourceBundle;
+
+public class RunConfigurationEditorController implements Initializable {
+    public JFXListView<RunConfiguration> savedConfigurationsList;
+    public JFXButton okButton, cancelButton, applyButton;
+    public JFXRippler addNewRunConfigurationButton, removeSelectedRunConfigurationButton;
+    public GridPane propertyGridPane;
+
+    public RunConfigurationEditorController() {}
+
+    @Override
+    public void initialize(URL location, ResourceBundle resources) {
+        initializeSavedConfigurationsList();
+    }
+
+    private void initializeSavedConfigurationsList() {
+        var runConfigsJson = HUPPAAL.preferences.get(RunConfigurationPreferencesKeys.ConfigurationsList, "[]");
+        var currentConfig = HUPPAAL.preferences.get(RunConfigurationPreferencesKeys.CurrentlySelected, "");
+        var gson = new Gson();
+        List<RunConfiguration> runConfigurations = gson.fromJson(runConfigsJson, RunConfiguration.listTypeToken);
+        savedConfigurationsList.getItems().addAll(runConfigurations);
+        // TODO: set that c is selected if c.isPresent()
+        // var c = savedConfigurationsList.getItems().stream().filter(e -> e.name().equals(currentConfig)).findAny();
```

Comment:
Is there a reason that this is not part of the PR? I think it would be nice to include it to avoid the configuration dropdown being "empty" when a configuration exists

Reply:
I missed this during my cleanup step 😁 it has been fixed in a7913f33f47bd77265c2a9c6298149fc91a63d30

Reply:
Fixed 😁